Cargo.toml

```diff
@@ -14,10 +14,12 @@ readme = "README.md"
 keywords = ["c2pa", "xmp", "metadata"]
 edition = "2018"
 rust-version = "1.61.0"
+homepage = "https://contentauthenticity.org"
```

Comment:
Should this be opensource.contentauthenticity.org?

Reply:
It could be, I debated about that. But the opensource page info for the sdk is just a link back to the readme that we already display. I thought it might be more useful to give people the larger context here.